src/app/pages/PhotoGalleryPage/__snapshots__/index.test.jsx.snap

```diff
@@ -2,61 +2,71 @@
 
 exports[`Photo Gallery Page should not show the pop-out timestamp when allowDateStamp is false 1`] = `
 <DocumentFragment>
-  .emotion-250 {
+  @keyframes animation-0 {
```

Comment:
Looks like this has happened due to the image components being altered and refreshing the snapshots? https://github.com/bbc/psammead/blob/latest/packages/components/psammead-image/src/index.jsx#L8